src/kernels/composable_kernel/include/utility/config.hpp

```diff
@@ -11,6 +11,23 @@
 // device backend
 #define CK_DEVICE_BACKEND_AMD 1
 
+// hardware instruction support list
+#ifndef CK_HARDWARE_SUPPORT_AMD_V_MAC_F32
+#define CK_HARDWARE_SUPPORT_AMD_V_MAC_F32 1
```

Comment:
AFAICS these macros are always defined by the host. Does it worth to #error instead of defining the default values? Alternatively, the defaults could be set to something that would work on all GPUs (if host code fails to define some of macros).

Reply:
I change these values to 0. Setting them to 1 does cause problem with compilation.